src/AzureExtension/Assets/QuickStartPlayground/docsEmbeddings-OpenAI.json

```diff
@@ -4639,1546 +4639,1546 @@
-            0.0028268506,
-            -0.007875543,
-            -0.023303263,
-            0.014436757,
-            -0.0013469264,
-            0.021300474,
-            0.030938888,
-            0.0090229735,
-            0.017002828,
-            -0.014248996,
-            -0.0091690095,
-            -0.022969464,
-            0.016001435,
-            0.00030560105,
-            0.006608154,
-            -0.006394315,
-            0.01181853,
-            -0.002030169,
-            0.0027720868,
-            0.045730304,
-            0.01472883,
-            -0.0028112037,
-            -0.03577896,
-            0.0077034286,
-            0.0053146877,
-            -0.013717005,
-            0.009810528,
-            -0.0007060607,
-            -0.013727437,
-            -0.004490624,
-            -0.018974321,
-            0.027934708,
-            0.007901621,
-            -0.018369313,
-            0.036676038,
-            0.026578654,
-            -0.0006160917,
-            0.032566153,
-            -0.0007503932,
-            -0.0038699687,
-            -0.015500737,
-            0.01861966,
-            -0.020601586,
-            0.004480193,
-            0.039471596,
-            0.043185096,
-            -0.04506271,
-            0.0061596134,
-            -0.007364415,
-            5.0240815e-05,
-            0.018796992,
-            0.0045375647,
-            -0.0027512244,
-            0.011964567,
-            0.05282351,
-            -0.036467414,
-            0.02795557,
-            0.002627354,
-            -0.0087830555,
-            0.016815066,
-            0.01110921,
-            0.020048734,
-            0.014843573,
-            0.021237887,
-            -0.050194852,
-            -0.0024826215,
-            0.024262931,
-            -0.009080345,
-            0.03360927,
-            0.018598799,
-            -0.0021957639,
-            -0.014676674,
-            -0.009294184,
-            0.011088347,
-            0.0026938529,
-            0.045813754,
-            -0.014634949,
-            -0.0040368675,
-            0.023261538,
-            -0.016679462,
-            -0.019151652,
-            -0.04226715,
-            0.029582834,
-            -0.0019128182,
-            -0.038407616,
-            0.0041802963,
-            -0.032065455,
-            0.039993156,
-            -0.008626588,
-            -0.00674376,
-            -0.0009909624,
-            0.0057319347,
-            -0.027642634,
-            0.033233747,
-            0.02689159,
-            -0.02866489,
-            -0.039763667,
-            -0.012580006,
-            -0.03517395,
-            0.0014694929,
-            -0.017263608,
-            0.015177371,
-            0.016533425,
-            0.0036300516,
-            0.06162743,
-            0.011046623,
-            -0.01657515,
-            -0.027475735,
-            -0.037260186,
-            -0.014457619,
-            -0.01217319
+            -0.027098652,

```

Comment:
Noting here that the embeddings vector changed in spite of the prompt remaining the same. The API can return slight variations (e.g., if OpenAI adds some fine-tuning or other changes to the underlying model); this doesn't materially change the overall semantic meaning of the embedding, and I verified that the 'before' and 'after' vectors have a miniscule distance (on the order of thousandths).